packages/framework-core/src/booster-read-model-dispatcher.ts

```diff
@@ -13,9 +13,6 @@ import {
 import { BoosterAuth } from './booster-auth'
 import { Booster } from './booster'
 
-// TODO: Think if this should be configurable
-const subscriptionDurationSeconds = 24 * 60 * 60 // 24 hours
```

Comment:
This was finally moved to the Booster config, so it is configurable by the user now (it has sensible defaults)